ballerina/http_client_connection_pool.bal

```diff
@@ -29,14 +29,22 @@ configurable int maxActiveStreamsPerConnection = 100;
 # + maxActiveStreamsPerConnection - Maximum active streams per connection. This only applies to HTTP/2. Default value is 100
 # + minEvictableIdleTime - Minimum evictable time for an idle connection in seconds. Default value is 5 minutes
 # + timeBetweenEvictionRuns - Time between eviction runs in seconds. Default value is 30 seconds
+# + minIdleTimeInStaleState - Minimum time in seconds for a connection to be kept open which has received a GOAWAY.
+#                             This only applies for HTTP/2. Default value is 5 minutes. If the value is set to -1,
+#                             the connection will not be closed until all existing streams are completed
+# + timeBetweenStaleCheck - Time between the connection stale check run in seconds. This only applies for HTTP/2.
+#                           Default value is 30 seconds
 public type PoolConfiguration record {|
     int maxActiveConnections = maxActiveConnections;
     int maxIdleConnections = maxIdleConnections;
     decimal waitTime = waitTime;
     int maxActiveStreamsPerConnection = maxActiveStreamsPerConnection;
     decimal minEvictableIdleTime = 300;
     decimal timeBetweenEvictionRuns = 30;
+    decimal minIdleTimeInStaleState = 300;
+    decimal timeBetweenStaleCheck = 30;
```

Comment:
I think we have to make them as configurable just like `maxActiveConnections` and `waitTime`

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-http/pull/1833/commits/feeca1fbb6a4e9b33ff2074d822c532a837c897c

Reply:
We need to add `minIdleTimeInStaleState` and `timeBetweenStaleCheck` as configurables, and set them as the default values
```suggestion
    decimal minEvictableIdleTime = minEvictableIdleTime;
    decimal timeBetweenEvictionRuns = timeBetweenEvictionRuns;
    decimal minIdleTimeInStaleState = minIdleTimeInStaleState;
    decimal timeBetweenStaleCheck = timeBetweenStaleCheck;
```